colossalai/zero/gemini/gemini_optimizer.py

```diff
@@ -86,6 +86,12 @@ def __init__(self,
         self.clipping_flag = clipping_norm > 0.0
         self.max_norm = clipping_norm
         self.verbose = verbose
+        self.untested_optim_flag = True if type(optim) not in _AVAIL_OPTIM_LIST else False
```

Comment:
I think this line can be simplified as `self.untested_optim_flag = type(optim) not in _AVAIL_OPTIM_LIST`

Reply:
Thanks for the correction. I immediately reflected.